spec.md

```diff
@@ -1071,6 +1286,15 @@ message GetCapacityRequest {
   // specific `parameters`. These are the same `parameters` the CO will
   // use in `CreateVolumeRequest`. This field is OPTIONAL.
   map<string, string> parameters = 2;
+  
+  // If specified, the Plugin SHALL report the capacity of the storage
+  // that can be used to provision volumes that satisfy ALL of the
+  // specified `accessibility_requirements`. These are the same
+  // `accessibility_requirements` the CO will use in a
+  // `CreateVolumeRequest`.
+  // This field is OPTIONAL. This field SHALL NOT be set unless the
+  // plugin advertises the ACCESSIBILITY_CONSTRAINTS capability.
+  TopologyRequirement accessibility_requirements = 7;
```

Comment:
What's the advantage of `TopologyRequirement` here vs. `Topology`? In other words, what benefit does an SP derive from seeing the CO's "preferred" topologies; would the reported capacity change based on the presence of the CO's preference?
Thinking about this RPC and `ValidateVolumeCapabilitiesRequest` and wondering why the topo accessibility fields are of different types.

Reply:
Good point, will change to Topology.